data/gmh/Frings_1976_MorantundGalie/metadata.txt

```diff
@@ -0,0 +1,48 @@
+publicationType : "book"
+publicationTitle : "Morant und Galie"
+publicationDate : "1976"
+publicationPlaces:
+    - publicationPlace: "Berlin"
+publicationSeries : "Deutsche Texte des Mittelalters"
+publicationNum: "69"
+publicationStemmaNum: "1"
+publicationAuthors:
+    - publicationAuthor: "Frings, Theodor"
+publicationPage : "XXXVI"
+publicationLink : ""
+workTitle : "Morant und Galie"
+workViaf : "220423428"
+workOrigDate : ""
+workOrigPlace : ""
+workAuthors:
+    - workAuthor: "Anonymous"
+      workAuthorViaf: ""
+workGenre : ""

```

Comment:
No Genre ?

Reply:
Oops. It's epic. Part of the stories around Charlemagne and descending from lost French sources.